config/relay/credentials.json

```diff
@@ -0,0 +1,5 @@
+{
+  "secret_key": "OxE6Du8quMxWj19f7YDCpIxm6XyU9nWGQJkMWFlkchA",
+  "public_key": "SMSesqan65THCV6M4qs4kBzPai60LzuDn-xNsvYpuP8",
+  "id": "aaa69140-a738-4e3b-80d7-0afe1f27e066"
```

Comment:
Can we perhaps make the id human readable or just less random?
So that we see right away that it's dev-only.

Reply:
ok changed to "id": "88888888-4444-4444-4444-cccccccccccc" (i.e. 8*8, 4*4, 4*4, 4*4, 12*12)